source/getting-started/autostart-systemd.markdown

```diff
@@ -17,10 +17,10 @@ $ ps -p 1 -o comm=
 
 If the preceding command returns the string `systemd`, you are likely using `systemd`.
 
-If you want Home Assistant to be launched automatically, an extra step is needed to setup `systemd`. You need a service file to control Home Assistant with `systemd`. If you are using a Raspberry Pi with Raspbian then replace the `[your user]` with `pi` otherwise use your user you want to run Home Assistant. `ExecStart` contains the path to `hass` and this may vary. Check with `whereis hass` for the location.
+If you want Home Assistant to be launched automatically, an extra step is needed to setup `systemd`. You need a service file to control Home Assistant with `systemd`. If you are using a Raspberry Pi with Raspbian then replace the `[your user]` and %1 with `pi` otherwise use your user you want to run Home Assistant. `ExecStart` contains the path to `hass` and this may vary. Check with `whereis hass` for the location.
 
 ```bash
-$ su -c 'cat <<EOF >> /etc/systemd/system/home-assistant@[your user].service
+$ sudo su -c 'cat <<EOF >> /etc/systemd/system/home-assistant@[your user].service
```

Comment:
Honestly, I would count myself a noob with this stuff.  I made the changes to get it to work.
You can see, I only made 2 changes...i) insert sudo.  This was required because I believe out of the box a raspberry pi does not have a su password...and if it did, I don't know what it is.  I was just following recommendations for other sites on the web.  Your suggestion could well be better than mine...but I can say that mine got it over the line.